Confidence/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -226,6 +237,7 @@ class Confidence internal constructor(
             eventSenderEngine.stop()
         }
         coroutineScope.cancel()
+        disableDebugLogger()
```

Comment:
Maybe we don't want to silence the logger explicitly? It should be come silent once everything is "closed" and if it doesn't that'd be valuable debugging info that something is broken.

Reply:
Good point!